torch/utils/optimizer.py

```diff
@@ -0,0 +1,26 @@
+"""
+This module contains utility method for mobile model optimization and lint.
+"""
+
+import io
+import torch
+
+
+def optimize_for_mobile(scripted_model):
```

Comment:
Also, I think it is important to point out that current optimization pass does the optimization `inplace`. Maybe we can change that so we return a new module.